src/renderers/dom/client/eventPlugins/ChangeEventPlugin.js

```diff
@@ -99,14 +99,24 @@ function runEventInBatch(event) {
 function startWatchingForChangeEventIE8(target, targetID) {
   activeElement = target;
   activeElementID = targetID;
-  activeElement.attachEvent('onchange', manualDispatchChangeEvent);
+
+  if (activeElement.attachEvent) {
+    activeElement.attachEvent('onchange', manualDispatchChangeEvent);
+  } else {
+    activeElement.addEventEventListener('change', manualDispatchChangeEvent, false);
```

Comment:
@syranide Prior to this change, this code path targeted IE8 and IE9.  This change makes IE10 and IE11 also follow the same code path.  IE11 does not have `attachEvent` which is why this is reachable.  See https://github.com/facebook/react/pull/3826#issuecomment-109784848 for context.